class/defaults.yml

```
@@ -63,6 +64,20 @@ parameters:
       enabled:
         # * Enables all alerts
         "*": false
+      custom:
+        # Adapted from https://github.com/rhobs/configuration/blob/70aa0baf3d5c7a5a1a3d2e5b2828b32220944b61/resources/observability/prometheusrules/observatorium-custom-metrics-production.prometheusrules.yaml#L14
+        ThanosNoStoreBlocksLoaded:
+          enabled: false
+          annotations:
+            description: Thanos Store {{$labels.namespace}}/{{$labels.job}} has not loaded any blocks in the last 3 hours.
+            message: Thanos Store {{$labels.namespace}}/{{$labels.job}} has not loaded any blocks in the last 3 hours.
+            runbook: https://hub.syn.tools/thanos/runbooks/ThanosNoStoreBlocksLoaded.html
+            summary: Thanos Store has not loaded any blocks in the last 3 hours.
+          expr: |
+            absent(thanos_bucket_store_blocks_last_loaded_timestamp_seconds) != 1 and (time() - thanos_bucket_store_blocks_last_loaded_timestamp_seconds) > 3 * 60 * 60
+          for: 10m
+          labels:
+            severity: warning
```

Do we really want to use the same flawed design as for the component config here, or would it make more sense to use a design that allows for further custom fields  in the future. e.g. using key `rule` for the contents of the alert rule:

```suggestion
          enabled: false
          rule:
            annotations:
              description: Thanos Store {{$labels.namespace}}/{{$labels.job}} has not loaded any blocks in the last 3 hours.
              message: Thanos Store {{$labels.namespace}}/{{$labels.job}} has not loaded any blocks in the last 3 hours.
              runbook: https://hub.syn.tools/thanos/runbooks/ThanosNoStoreBlocksLoaded.html
              summary: Thanos Store has not loaded any blocks in the last 3 hours.
            expr: |
              absent(thanos_bucket_store_blocks_last_loaded_timestamp_seconds) != 1 and (time() - thanos_bucket_store_blocks_last_loaded_timestamp_seconds) > 3 * 60 * 60
            for: 10m
            labels:
              severity: warning
```

---

Yes this is clearly the better idea. Let's not repeat the mistakes.